spotify/views.py

```diff
@@ -13,7 +13,7 @@
 
 
 def session_cache_path(request):
-    print(settings.CACHES_FOLDER)
+    # print(settings.CACHES_FOLDER)
```

Comment:
Here's the rule I propose for commented out code in code-reviews. EITHER it is removed OR it is accompanied by an explanation as to why it is being retained AND ADDITIONALLY it is never commented out if we can instead use this preferable idiom:
```
def session_cache_path(request):
    if False: print(settings.CACHES_FOLDER)  # Handy to debug when <<INSERT_EXPLANATION_HERE>>.
    return f'{settings.CACHES_FOLDER}/{request.user.email}'
```
This idiom has the advantage that the code is passed through the early compiler pipelines into the more analytical bits of the compiler. This means that when the code bit-rots the compiler gets a chance to warn you.

Reply:
Fair enough. I should have just deleted it. Will remove it now.